### ISSUE TITLE
fatal: unknown date format

### ISSUE THREAD

**Issue by fredrikaverpil** [fredrikaverpil/pyside2-linux#3]
https://travis-ci.org/fredrikaverpil/pyside2-linux/jobs/335032063#L862

```
fatal: unknown date format format-local:%Y-%m-%dT%H:%M:%S+00:00
```

**Comment by fredrikaverpil** [comment]
Seems like a git issue.

```
# centos7-qt5.9... git version
Found Git: /usr/bin/git (found version "1.8.3.1")
```

**Comment by fredrikaverpil** [comment]
Instructions for CentOS 6, 7 solved this: https://stackoverflow.com/a/38133865

```
# CentOS 7 instructions:
$ sudo yum install https://centos7.iuscommunity.org/ius-release.rpm
$ sudo yum erase git
$ sudo yum install epel-release 
$ sudo yum install git2u
```

```
# CentOS 6 instructions:
$ sudo yum install https://centos6.iuscommunity.org/ius-release.rpm
$ sudo yum erase git
$ sudo yum install epel-release
$ sudo yum install git2u
```

**Comment by fredrikaverpil** [comment]
Fixed in 3db23a02b81d848a6b8a8fdaff1ed9572e130566
